projects/media-viewer/src/lib/media-viewer/viewers/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -66,6 +69,22 @@ export class PdfViewerComponent implements AfterViewInit, OnDestroy {
     }
   }
 
+  rotateSinglePage(rotateSinglePageOperation: RotateSinglePageOperation) {
+    if (this.pdfViewer) {
+      const pageView = this.pdfViewer.getPageView(rotateSinglePageOperation.pageIndex);
+      if (pageView) {
+        let currentPageRotation = pageView.rotation;
+        if (rotateSinglePageOperation.direction === RotateDirection.LEFT) {
+          currentPageRotation = (currentPageRotation - 90) % 360;
+        } else if (rotateSinglePageOperation.direction === RotateDirection.RIGHT) {
+          currentPageRotation = (currentPageRotation + 90) % 360;
+        }
```

Comment:
why not use a numeric value when invoking rotation (+90, -90), so that we don't have to check for the direction and would just add it